src/main/java/com/porpapps/wealthwatchapi/model/Portfolio.java

```diff
@@ -0,0 +1,16 @@
+package com.porpapps.wealthwatchapi.model;
+
+import lombok.Data;
+
+import java.time.LocalDate;
+
+@Data
+public class Portfolio {
+    private Integer id;
+    private Integer userId;
```

Comment:
Use the class User instead of the id here and for the other "foreign keys" relationships

Reply:
done